backend/src/schema/resolvers/searches.spec.js

```diff
@@ -26,6 +26,8 @@ beforeAll(async () => {
 
 afterAll(async () => {
   await cleanDatabase()
+  driver.close()
+  neode.close()
```

Comment:
whats that?

Reply:
driver is the driver of the neo4j library
neode is the driver of the neode library
Can we have a follow PR for the closing in general issue? I have no idea how to do this so far, and it looks a bit unrealistic to achieve this